src/app/routes/uci.js

```diff
@@ -37,23 +37,23 @@ module.exports = function (app) {
 
 function addHeaders() {
     return function (proxyReqOpts, srcReq) {
-    //    let decoratedHeaders =  proxyUtils.decorateRequestHeaders(discussions_middleware)()
+    //    let decoratedHeaders =  proxyUtils.decorateRequestHeaders(uci_service_base_url)()
         proxyReqOpts.headers['Authorization'] = 'Bearer ' + srcReq.session['nodebb_authorization_token'];
```

Comment:
@ChakshuGautam No need to pass nodebb_authorization_token this token is for discussion middleware service

Reply:
Updated